mirage-flow: Package `oUnit' not found

My installation failed with
```
∗  installed conf-gmp.1.1
∗  installed cmdliner.0.9.8
∗  installed camlzip.1.05
∗  installed base64.2.0.0
∗  installed dolog.1.1
∗  installed fieldslib.113.00.00
∗  installed alcotest.0.4.5
∗  installed mirage-no-xen.1
∗  installed hex.0.2.0
∗  installed io-page.1.5.1
∗  installed magic-mime.1.0.0
∗  installed mirage-types.2.6.0
∗  installed mirage-types-lwt.2.6.0
∗  installed mstruct.1.3.2
[ERROR] The compilation of mirage-flow failed at "./configure
        --prefix=/Users/djs/go/src/github.com/docker/pinata/v1/db/irmin/_build/root/4.02.3
        --enable-tests".
∗  installed mirage-profile.0.5
∗  installed channel.1.0.0
∗  installed zarith.1.3
∗  installed ocamlgraph.1.8.6
∗  installed ounit.2.0.0
∗  installed crc.0.9.0
∗  installed dns.0.15.3
∗  installed mirage-dns.2.5.0
∗  installed nocrypto.0.5.1
∗  installed conduit.0.8.8
∗  installed mirage-conduit.2.2.0
∗  installed cohttp.0.19.3
∗  installed mirage-http.2.5.1
```
and
```
#=== ERROR while installing mirage-flow.1.0.3 =================================#
# opam-version 1.2.2
# os           darwin
# command      ./configure --prefix=/Users/djs/go/src/github.com/docker/pinata/v1/db/irmin/_build/root/4.02.3 --enable-tests
# path         /Users/djs/go/src/github.com/docker/pinata/v1/db/irmin/_build/root/4.02.3/build/mirage-flow.1.0.3
# compiler     4.02.3
# exit-code    1
# env-file     /Users/djs/go/src/github.com/docker/pinata/v1/db/irmin/_build/root/4.02.3/build/mirage-flow.1.0.3/mirage-flow-11947-34169e.env
# stdout-file  /Users/djs/go/src/github.com/docker/pinata/v1/db/irmin/_build/root/4.02.3/build/mirage-flow.1.0.3/mirage-flow-11947-34169e.out
# stderr-file  /Users/djs/go/src/github.com/docker/pinata/v1/db/irmin/_build/root/4.02.3/build/mirage-flow.1.0.3/mirage-flow-11947-34169e.err
### stderr ###
# ocamlfind: Package `oUnit' not found
# W: Field 'pkg_ounit' is not set: Command ''/Users/djs/go/src/github.com/docker/pinata/v1/db/irmin/_build/root/4.02.3/bin/ocamlfind' query -format %d oUnit > '/var/folders/ml/fqqlxl6j03l2lf9ljfms9m540000gn/T/oasis-8c1409.txt'' terminated with error code 2
# E: Cannot find findlib package oUnit
# E: Failure("1 configuration error")
```
It looks like `--enable-tests` was activated by alcotest being installed, but the tests require oUnit too, which was installed afterwards.

--- a/packages/mirage-flow/mirage-flow.1.0.3/opam
+++ b/packages/mirage-flow/mirage-flow.1.0.3/opam
@@ -6,7 +6,7 @@ bug-reports: "https://github.com/mirage/mirage-flow/issues/"
 license: "ISC"
 dev-repo: "https://github.com/mirage/mirage-flow.git"
 build: [
-  ["./configure" "--prefix=%{prefix}%" "--%{alcotest:enable}%-tests"]
+  ["./configure" "--prefix=%{prefix}%" "--%{alcotest+ounit:enable}%-tests"]
   [make]
 ]
 build-test: [make "test"]

